class_vimrc.txt

```diff
@@ -19,8 +19,8 @@ set incsearch               "highlight while typing
 
 set hidden                  "allow modified buffers to hide
 
-set number                  "Display line numbers incrementing
-"set number relativenumber  "Display line numbers in relative positions
+set number relativenumber   "Display line numbers in relative positions
+"set number                 "Display line numbers incrementing
```

Comment:
You actually need both set.  With just relative number, the cursor position is labeled with `0`.  With both, the current line has the line number while the rest are relative.